### DISCUSSION TITLE
GitHub student pack migration to an other GitHub account

### DISCUSSION THREAD

**Discussion by BenjedouNaim** [community/community#134844] [GitHub Education]
### Hi everyone,

I want to restart my account from 0 so I created an other one and I want to migrate my GitHub student benefits to the other account..can I do that ?

**Comment by davevad93** [comment]
Hi @Nviim ,

It's not possible to transfer the `Student Developer Pack` benefits from one account to another, you can merge two accounts but the account with the benefits will still be the main one associated with GitHub Education. For more info about merging accounts see [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts).
